bitmovin-analytics-datasource/src/utils/licenses.ts

```diff
@@ -0,0 +1,71 @@
+import { lastValueFrom } from 'rxjs';
+import { getBackendSrv } from '@grafana/runtime';
+import { SelectableValue } from '@grafana/data';
+
+type AnalyticsLicense = {
+  readonly name: string;
+  readonly id: string;
+  readonly licenseKey?: string;
```

Comment:
why optional ? I think our licenses from backend always have `licenseKey` field set ?

Reply:
Virtual and demo license do not have a license-key but only the idea, that's why I marked it as optional.